stm32_base/hal/CAN_stm32/CAN_device_stm32.c

```diff
@@ -32,34 +32,50 @@ static xQueueHandle xCan2Rx = NULL;
  100k:    12  8   1   20 */
 
 #define CAN_BAUD_COUNT 5
-static const u8 can_baud_bs1[] = { CAN_BS1_12tq, CAN_BS1_12tq, CAN_BS1_8tq,	CAN_BS1_8tq, CAN_BS1_12tq };
-static const u8 can_baud_bs2[] = { CAN_BS1_8tq, CAN_BS1_8tq, CAN_BS1_5tq, CAN_BS1_5tq, CAN_BS1_8tq };
-static const u8 can_baud_sjw[] = { CAN_SJW_1tq, CAN_SJW_1tq, CAN_SJW_1tq, CAN_SJW_1tq, CAN_SJW_1tq };
+static const u8 can_baud_bs1[] =
+    { CAN_BS1_12tq, CAN_BS1_12tq, CAN_BS1_8tq, CAN_BS1_8tq, CAN_BS1_12tq };
+
+static const u8 can_baud_bs2[] =
+    { CAN_BS1_8tq, CAN_BS1_8tq, CAN_BS1_5tq, CAN_BS1_5tq, CAN_BS1_8tq };
+
+static const u8 can_baud_sjw[] =
+    { CAN_SJW_1tq, CAN_SJW_1tq, CAN_SJW_1tq, CAN_SJW_1tq, CAN_SJW_1tq };
+
 static const u8 can_baud_pre[] = { 20, 16, 12, 6, 2 };
 static const u32 can_baud_rate[] = { 100000, 125000, 250000, 500000, 1000000 };
 
-static int initQueues() {
+static int initQueues()
+{
 	int success = 1;
 
-	if (! (xCan1Rx && xCan1Tx && xCan2Rx && xCan2Tx)){
+	if (!(xCan1Rx && xCan1Tx && xCan2Rx && xCan2Tx)) {
 		xCan1Rx = xQueueCreate(CAN_QUEUE_LENGTH,
-				( unsigned portBASE_TYPE ) sizeof( CanRxMsg ));
+				       (unsigned portBASE_TYPE)
+				       sizeof(CanRxMsg));
 		xCan1Tx = xQueueCreate(CAN_QUEUE_LENGTH + 1,
-				( unsigned portBASE_TYPE ) sizeof( CanRxMsg ));
+				       (unsigned portBASE_TYPE)sizeof(CanRxMsg));
+		if (xCan1Rx == NULL || xCan1Rx == NULL) {
+			success = 0;
+			goto cleanup_and_return;
```

Comment:
Goto is find here, but why not simply return 0 here?  Seems like goto is unnecessary.